chunky3d/helpers.py

```diff
@@ -42,6 +42,7 @@ def slice_normalize(key, shape):
     :return: tuple of 'normalized' slices
     :raises: IndexError if slice start, stop or step is not an integer"""
 
+    shape = tuple([int(x) for x in shape])
```

Comment:
Sorry, but I don't see exactly how this could happen - slice_normalize() usually gets `shape` from `chunky3d.Sparse`'s `.shape` field, which is a tuple of `ints`.

What is the use case which passes an array of `dtype == uint`? Can it happen after some "reshape" of the Sparse object?